### PR TITLE
Support running Ambari as non-root

### PR DESCRIPTION
**Pull request by adoroszlai**

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [defaults]
+inventory = hosts
```
Comment (oleewere) [inline ansible.cfg:2]: why is that required?

File: playbooks/install-ambari-2_6.yml [hunk 2]
```diff
@@ -15,12 +15,16 @@
   become: true
   tasks:
     - include_tasks: ../roles/ambari-server/tasks/install-{{ ansible_os_family }}.yml
-    - import_tasks: ../roles/ambari-server/tasks/setup.yml
+    - import_tasks: ../roles/ambari-server/tasks/sudo.yml
```
Comment (oleewere) [inline playbooks/install-ambari-2_6.yml:18]: although we setting the sudoers... should we use become_user for every other tasks or playbook with remote_ambari_user? (like for ambari server start etc.)

File: ansible.cfg (hunk 1 above)
Comment (adoroszlai) [reply ansible.cfg:2]: It's not directly related to the non-root change.  It's just a default, so you don't have to specify `-i hosts` for each command.  It can be overridden though.

I can submit a one-liner PR for this separately, if needed.

File: playbooks/install-ambari-2_6.yml (hunk 2 above)
Comment (adoroszlai) [reply playbooks/install-ambari-2_6.yml:18]: Connecting to hosts as non-root user and becoming another non-root user [poses problems](http://docs.ansible.com/ansible/latest/become.html#becoming-an-unprivileged-user).  There are various solutions suggested in the docs, but the only one that worked for me on all hosts (CentOS, Debian, Ubuntu) is connecting as root.  If that's acceptable, I can make this change (set `ansible_ssh_user=root` and add `become_user`).

**Comment by oleewere** [comment]
@adoroszlai , i will me, but then i will revert the ansible.cfg change + later i can add the become_user stuff
